lib/index.js

```diff
@@ -122,13 +147,13 @@ exports.Model = class SchwiftyModel extends Model {
         let validation = this.tryValidate(joiSchema, json);
```

Comment:
Can `this.tryValidate()` be a `static`, or even an `internals`?  Is it called anywhere else?